tests/tx/test_indexes.py

```diff
@@ -609,6 +609,23 @@ def test_addresses_index_empty(self):
         self.assertTrue(addresses_indexes.is_address_empty(address))
         self.assertEqual(addresses_indexes.get_sorted_from_address(address), [])
 
+    def test_addresses_index_last(self):
+        from hathor.wallet.util import generate_multisig_address, generate_multisig_redeem_script
+
+        addresses_indexes = self.manager.tx_storage.indexes.addresses
+        # XXX: since we didn't add any multisig address, this is guaranteed to be reach the tail end of the index
+        assert settings.P2PKH_VERSION_BYTE[0] < settings.MULTISIG_VERSION_BYTE[0]
+
+        # generating a multisig address:
+        address = generate_multisig_address(generate_multisig_redeem_script(2, [
+            bytes.fromhex('0250bf5890c9c6e9b4ab7f70375d31b827d45d0b7b4e3ba1918bcbe71b412c11d7'),
+            bytes.fromhex('02d83dd1e9e0ac7976704eedab43fe0b79309166a47d70ec3ce8bbb08b8414db46'),
+        ]))
+        assert address is not None
+
+        self.assertTrue(addresses_indexes.is_address_empty(address))
```

Comment:
Maybe also check for the address `b'\xff' * 34` as well since it's guaranteed to reach the end of the records in the index.

Reply:
Done, but I'm using `'\x7f' * 34` instead, basically because the method takes an `str` and `\xff` is out of the range of what can be encoded to bytes.

Reply:
Got it! Great!